src/translations.js

```diff
@@ -722,7 +739,8 @@ const wakatimeCardLocales = {
     vi: "Năm ngoái",
     se: "Förra året",
     th: "ปีที่แล้ว",
-    sr: "Претходна година",
+    "sr-Cyrl": "Прошла год.",
```

Comment:
Label is shortened because it goes out of bounds:
<img width="440" height="309" alt="github-readme-stats serbian" src="https://github.com/user-attachments/assets/333677c3-d85a-4d8b-a477-8ad133b32173" />